app/assets/stylesheets/application.scss

```diff
@@ -506,3 +510,28 @@ p {
   cursor: pointer;
   color: blue;
 }
+
+/*
```

Comment:
This file is already very large, so this should probably go to `moderator_applications.scss`, because it's only used there anyway. Knowing what views use what CSS would make it a lot easier to clean up later. Though on the other hand, this isn't inherently related to the moderator applications view. Though everything's probably going to be ripped up with the redesign anyway.

Reply:
Yeah, it's fine to leave this as is.